pkg/kev/skaffold.go

```diff
@@ -109,22 +106,78 @@ func LoadSkaffoldManifest(path string) (*SkaffoldManifest, error) {
 	return s, yaml.Unmarshal(data, &s)
 }
 
-// AddProfiles injects kev profiles to existing Skaffold manifest
+// InjectProfiles injects kev profiles to existing Skaffold manifest
 // Note, if profile name already exists in the skaffold manifest then profile won't be added
-func AddProfiles(path string, envs []string, includeAdditional bool) (*SkaffoldManifest, error) {
+func InjectProfiles(path string, envs []string, includeAdditional bool) (*SkaffoldManifest, error) {
 	skaffold, err := LoadSkaffoldManifest(path)
 	if err != nil {
 		return nil, err
 	}
 
 	skaffold.SetProfiles(envs)
 	if includeAdditional {
-		skaffold.AdditionalProfiles()
+		skaffold.SetAdditionalProfiles()
 	}
 
 	return skaffold, nil
 }
 
+// UpdateSkaffoldBuildArtifacts updates skaffold build artefacts with freshly discovered list of images and contexts.
+// Note, it'll persist updated build artefacts in the skaffold.yaml file only when change in build artefacts was detected.
+// Important: The last discovered images and contexts will be persisted (if changed)!
+func UpdateSkaffoldBuildArtifacts(path string, project *ComposeProject) error {
```

Comment:
I wonder if the this skaffold specific stuff should go on a package which encapsulates all the skaffold specifics

Reply:
May become part of future PR.